src/association/association_internal.rs

```diff
@@ -797,6 +800,11 @@ impl AssociationInternal {
                         self.use_forward_tsn = true;
                     }
                 }
+            } else {
+                if let Some(_) = param.as_any().downcast_ref::<ParamForwardTsnSupported>() {
+                    log::debug!("[{}] use ForwardTSN (on initAck)", self.name);
+                    self.use_forward_tsn = true;
+                }
```

Comment:
I've a question about the previous logic: can `Forward-TSN` support be communicated via the `ParamSupportedExtensions` param? I couldn't find any confirmation in the IETF RFCs on that... It seems like it's valid to communicate only `I-Forward-TSN` this way, which is a different extension.